oak-security-spi/src/test/java/org/apache/jackrabbit/oak/spi/security/user/util/UserUtilTest.java

```diff
@@ -246,7 +275,7 @@ public void testGetAuthorizableRootPathNullType() {
     }
 
 
-    @Test(expected = NullPointerException.class)
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
why is this change needed?

Reply:
the method was updated in 2018 and got a @NotNull param annotation added